tox.ini

```diff
@@ -0,0 +1,27 @@
+[tox]
+envlist =
+    py27,py34,py35,py36,py37,pypy,pypy3,coverage
+
+[testenv]
+commands =
+    zope-testrunner --test-path=. []
+deps =
+    .[test]
+
+[testenv:pypy3]
+commands =
+    zope-testrunner --test-path=. []
+deps =
+    .[test]
```

Comment:
You do not need to repeat these settings: they're automatically inherited from the base `[testenv]` section.  All you need is overrides.

Reply:
I changed that.